sdk/src/main/java/com/hedera/hashgraph/sdk/TransactionResponse.java

```diff
@@ -79,6 +93,15 @@ public TransactionRecord getRecord(Client client) throws TimeoutException, Prech
             .execute(client);
     }
 
+    public TransactionRecord forceGetRecord(Client client) throws TimeoutException, PrecheckStatusException {
+        forceGetReceipt(client);
```

Comment:
You don't need to get receipt here.

The only reason we `getReceipt()` in `getRecord()` is because `TransactionReceiptQuery` is free, where `TransactionRecordQuery` is _not_ free.  The intended behavior of `getRecord()` was to throw an exception if the status is not `SUCCESS`, so if we can determine that the status is not `SUCCESS` by using a free query, it makes sense to do that and save the user from paying for a record query in that case.

For `forceGetRecord()`, it doesn't make sense to check the receipt before getting the record, since this is never going to throw the exception; the user wants to pay for the `TransactionRecordQuery` and get the record regardless of the status.  You're basically just getting the receipt and then immediately throwing it away without doing anything with it.